### PR TITLE
MAT-7994: Add Completion Indicator On Population Criteria Sub Tabs

### PR DESCRIPTION
**Pull request by chubert-sb**
## MADiE PR

Jira Ticket: [MAT-7994](https://jira.cms.gov/browse/MAT-7994)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx [hunk 1]
```diff
@@ -796,24 +798,61 @@ const MeasureGroups = (props: MeasureGroupProps) => {
...
+                            ) : (
+                              <CheckCircleIcon
+                                sx={{
+                                  color: "#4D7E23",
+                                  marginRight: "10px",
+                                  height: "20px",
+                                  width: "20px",
+                                }}
+                              />
+                            )}
```
Comment (adongare) [inline src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx]: as we are using these icons in multiple components, does it make sense to introduce separate components(like CompleteIndicator and IncompleteIndicator, name can be anything)?

Comment (adongare) [reply src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx]: more usage- [src/components/editMeasure/details/EditMeasureDetailsSideNav.tsx](https://github.com/MeasureAuthoringTool/madie-measure/pull/737/files#diff-06f2fe6ff0e2aa831911a341f71005eb594366f9e54535d37088f8042d25d121)

Comment (chubert-sb) [reply src/components/editMeasure/populationCriteria/groups/QDM/QDMMeasureGroups.tsx]: That's a good point. I moved that chunk into its own component

**Review by sb-cecilialiu — APPROVED** [review]
